lib/beaker/hypervisor/vmpooler.rb

```diff
@@ -19,6 +19,18 @@ def initialize(vmpooler_hosts, options)
       @options = options
       @logger = options[:logger]
       @hosts = vmpooler_hosts
+      @credentials = load_credentials(@options[:dot_fog])
```

Comment:
> I'll work on this and update the PR when I have it functioning as desired.

Updated PR.  Manually tested `.fog` with `:vmpooler_token`, `.fog` without `:vmpooler_token`, and no `.fog`.